sys/fido2/ctap/transport/hid/Kconfig

```diff
@@ -4,13 +4,18 @@
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 
-menuconfig KCONFIG_USEMODULE_FIDO2_CTAP_TRANSPORT_HID
+menuconfig MODULE_FIDO2_CTAP_TRANSPORT_HID
     bool "FIDO2 CTAP TRANSPORT HID"
-    depends on USEMODULE_FIDO2_CTAP_TRANSPORT_HID
+    select MODULE_ISRPIPE
+    select MODULE_ISRPIPE_READ_TIMEOUT
```

Comment:
This should not be needed, USBUS selects TSRB that selects these

Reply:
Removing both gave the error:
```txt
[genconfig.py]:ERROR-Treating Kconfig warnings as errors:
[genconfig.py]:ERROR-=> warning: MODULE_ISRPIPE_READ_TIMEOUT (defined at /home/b0re42/Uni/BachelorArbeit/RIOT/sys/isrpipe/Kconfig:15) has direct dependencies MODULE_ISRPIPE with value n, but is currently being y-selected by the following symbols:
 - MODULE_USBUS_HID (defined at /home/b0re42/Uni/BachelorArbeit/RIOT/sys/usb/usbus/hid/Kconfig:8), with value y, direct dependencies TEST_KCONFIG && MODULE_USBUS (value: y), and select condition TEST_KCONFIG && MODULE_USBUS (value: y)
```

Based on this error only removing `MODULE_ISRPIPE_READ_TIMEOUT` works. 
Thanks !